src/lib/storage/table.hpp

```diff
@@ -225,7 +227,16 @@ class Table : private Noncopyable {
   const std::vector<ColumnID>& value_clustered_by() const;
   void set_value_clustered_by(const std::vector<ColumnID>& value_clustered_by);
 
+  /*
+   * Persist table to use mmap-based storage for its data.
+   * TODO: Explanation.
+   */
+
+  void persist();
```

Comment:
Related to Martin's comment about not being responsible for persisting a table in the `Table` class: `persist_table` in the storage manager might be an alternative.